docs/age_gender_guide.md

```diff
@@ -13,6 +13,13 @@ curl --create-dirs https://download.01.org/opencv/2021/openvinotoolkit/2021.1/op
 
 ```
 
+Grant proper access rights to downloaded files
```

Comment:
Are you sure it's going to be required always?
If not, then perhaps you could add: _"You may need to..."_ to the beginning of that sentence?